pylib/cassandra-cqlsh-tests.sh

```diff
@@ -50,9 +50,6 @@ fi
 if [[ -z "${java_version}" ]]; then
   java_version=$(java -version 2>&1 | awk -F '"' '/version/ {print $2}' | sed 's/^1\.//g' | awk -F. '{print $1}')
 fi
-if [[ ${java_version} -ge 11 ]]; then
```

Comment:
Testing will show whether this is enough and whether CI is affected in some way by this change

Reply:
I am not sure which the new CCM is, but so far I did not see any problems.